packages/data-point/lib/reducer-types/reducer-list/factory.test.js

```diff
@@ -105,4 +105,28 @@ describe('factory#create', () => {
     expect(result.reducers[2].type).toBe('ReducerPath')
     expect(result.reducers[3].type).toBe('ReducerFunction')
   })
+
+  test('factory#throw error if reducer is false', () => {
+    expect(() => factory.create(createReducer, [false])).toThrow()
+  })
```

Comment:
> can we use toThrowErrorMatchingSnapshot instead of toThrow? that way, we can test to be sure that it's throwing an error with a useful message (the error message should be Invalid reducer type. Could not find a matching reducer type while parsing the value, but I think the current message is Cannot read property 'Symbol(reducer stub)' of undefined)

👍 In fact, it'd be great to have errors snapshotted across the tests.

Reply:
Great idea, I just added snapshots with the proper error messages.